src/ui/component-repository/CategoryFilter.js

```diff
@@ -2,24 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl, intlShape } from 'react-intl';
 import { reduxForm } from 'redux-form';
-import CheckboxGroup from 'ui/digital-exchange/common/CheckboxGroup';
-import SidebarFilter from 'ui/digital-exchange/common/SidebarFilter';
+import CheckboxGroup from 'ui/component-repository/common/CheckboxGroup';
+import SidebarFilter from 'ui/component-repository/common/SidebarFilter';
 
-const CategoryFilterBody = ({ digitalExchangeCategories, intl, onChange }) => (
+const CategoryFilterBody = ({ componentRepositoryCategories, intl, onChange }) => (
   <SidebarFilter
     title={intl.formatMessage({ id: 'digitalExchange.sidebar.categoryFilterTitle' })}
```

Comment:
I think this needs to be changed to: `componentRepository.sidebar.categoryFilterTitle`